packages/react-components/src/components/Picker/Picker.spec.tsx

```diff
@@ -7,24 +7,31 @@ import { IPickerProps, Picker } from './Picker';
 // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
I would expect some test that would make sure that a `single-select` is able to pick only one value (eg. by clicking two different options and making sure only the last one is triggered in `onSelect`. Another idea is making sure that `unselect` is working. 

In general I believe we could come up with more testing cases.